banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py

```diff
@@ -145,3 +145,28 @@ def get_customer_portal_url() -> str:
 	Returns the customer portal URL.
 	"""
 	return Admin().get_customer_portal_url()
+
+
+@frappe.whitelist()
+def get_app_health() -> Dict:
+	"""
+	Returns the app health.
+	"""
+	from frappe.utils.change_log import check_release_on_github
+	from frappe.utils.scheduler import is_scheduler_inactive
+
+	messages = {}
+	current_app_version = frappe.get_attr("banking.__version__")
+
+	latest_app_version = str(check_release_on_github("banking")[0])
+	if latest_app_version != current_app_version:
```

Comment:
- Shouldn't this be `<`? (convert `current_version` to a `Version` object first)
- The latest release should probably be specific to the current branch. (A v15 release won't be of much use for a v14 user -- applies to ERPNext as well, btw)

Reply:
> applies to ERPNext as well, btw
Do you suggest i do this check for erpnext too ?

Reply:
If it's easier to fix there, it wouldn't hurt :D

Reply:
An issue could be due to frappe being outdated too. Let's stick to banking. I think we're entering devops territory 😄